### ISSUE TITLE
OpenFeintSettingDevelopmentMode

### ISSUE THREAD

**Issue by macieks** [marmalade/openfeint#5]
With new OpenFeint SDK they've introduced new setting "OpenFeintSettingDevelopmentMode".

Unfortunately, this is mandatory, otherwise you get a dialog box "OpenFeint Developer Warning" at startup saying "You must specify a value for OpenFeintSettingDevelopmentMode in your settings passed to [OpenFeint initializeWithProducKey]".

I tried setting OpenFeintSettingDevelopmentMode to 0 or 1 (not sure how native enums translate to ints) but I still got that dialog box at startup (before logging in).

Any idea on how I could fix this? Or would it require library changes?
Thanks!

(and btw. great library!)

**Comment by faizann** [comment]
Did you try using the latest code from github? There is no such prompt in latest code. 

This situation is already handled inside extension code as

``` objective-c
#ifdef DEBUG
                 [NSNumber numberWithInt:OFDevelopmentMode_DEVELOPMENT], OpenFeintSettingDevelopmentMode,
#else
                 [NSNumber numberWithInt:OFDevelopmentMode_RELEASE], OpenFeintSettingDevelopmentMode,
#endif
                 [NSNumber numberWithInt:0], OpenFeintSettingDevelopmentMode,
```

Looking at this I realised that there is bug in my code where DevelopmentMode is always set to 0. I will delete this line and push to github later today.

**Comment by macieks** [comment]
Thanks for looking at it.

And yes, I did get latest code from github and yet I still see that dialog box on my ipad.

**Comment by faizann** [comment]
Hi

Forgot to inform you that I updated code a while ago and new libs are also in repo. This should surely get rid of developmentmode messagebox.

**Comment by macieks** [comment]
Thanks faizann. Works great now.

**Comment by macieks** [comment]
This is just FYI.

I realized I wasn't testing the correct version. Now that I did I could still see this popup message in my app.

However, I found that it's not showing in the example app, only in my app, so I tried to spot the difference in my initialization code. Eventually I found that in my app I was not setting OpenFeintSettingShortDisplayName property. And for some strange reason this is always forcing the popup dialog to show in OpenFeint.
